query/src/main/java/org/infinispan/query/impl/SearchManagerImpl.java

```diff
@@ -63,10 +60,10 @@ public SearchManagerImpl(AdvancedCache<?, ?> cache) {
    }
 
    @Override
-   public <E> CacheQuery<E> getQuery(String queryString, IndexedQueryMode indexedQueryMode, Class<?>... classes) {
+   public <E> CacheQuery<E> getQuery(String queryString, IndexedQueryMode indexedQueryMode) {
```

Comment:
It's a bit odd we loose the ```classes``` argument in SearcManager.getQuery but QueryEngine.buildCacheQuery still takes an array of classes